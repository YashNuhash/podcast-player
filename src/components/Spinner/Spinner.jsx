import React from 'react'
import styles from './Spinner.module.scss'

export const Spinner = () => (
    <div className={styles.SpinnerWrapper}>
        <span className={styles.Spinner}>
            <span className={styles.SpinItem} />
            <span className={styles.SpinItem} />
            <span className={styles.SpinItem} />
            <span className={styles.SpinItem} />
        </span>
    </div>
)

export default Spinner
