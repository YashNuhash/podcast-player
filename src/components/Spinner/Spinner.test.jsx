import React from 'react'
import { render } from '@testing-library/react'
import { Spinner } from './Spinner'

test('should render Spinner component', () => {
    const { getByTestId } = render(<Spinner />)
    expect(getByTestId('spinner')).toMatchSnapshot()
})
