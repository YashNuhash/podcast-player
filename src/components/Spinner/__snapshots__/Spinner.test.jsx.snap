// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Spinner component 1`] = `
<div
  class="SpinnerWrapper"
  data-testid="spinner"
>
  <span
    class="Spinner"
  >
    <span
      class="SpinItem"
    />
    <span
      class="SpinItem"
    />
    <span
      class="SpinItem"
    />
    <span
      class="SpinItem"
    />
  </span>
</div>
`;
